docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -2,11 +2,7 @@
  "cells": [
```

Comment:
<p>Removed the versions because it shows up as "dirty+...", and also we have versions in our docs anyway</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/6225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>